packages/eslint-plugin/configs/jsdoc.js

```diff
@@ -104,8 +104,17 @@ module.exports = {
 		'jsdoc/require-jsdoc': 'off',
 		'jsdoc/require-param-description': 'off',
 		'jsdoc/require-returns': 'off',
+		'jsdoc/require-yields': 'off',
 		'jsdoc/check-access': 'error',
 		'jsdoc/check-alignment': 'error',
+		'jsdoc/check-line-alignment': [
```

Comment:
This new addition seems to warrant a changelog entry

Reply:
Makes sense! Added here: https://github.com/WordPress/gutenberg/pull/25300/commits/2bf1f088acc6eeacc4de6f72e6b48b7379f508e0